### ISSUE TITLE
Add summary of recent payments

### ISSUE THREAD

**Issue by Lulcaz** [datasektionen/cashflow#212]
It would be nice to be able to see recent payments listed with the same format as in the bank.
<img width="657" alt="Summering Överföringar_" src="https://github.com/user-attachments/assets/81149479-d3fd-4e6a-b305-58ff3e4bb169">

**Comment by hfjallborg** [comment]
@Lulcaz is this something we still want?

**Comment by Lulcaz** [comment]
Yes, would be useful to discover/avoid/fix incorrect payments. An admin page like the one for listing verifications would probably be the best solution.
Relevant information that should be displayed:
Payment-ID (Data1337),
Date,
Paid by (who made thetransaction),
Paid to (receiver's name and/or accountnumber),
Amount

**Comment by hfjallborg** [comment]
Added with #405
